### ISSUE TITLE
[BUG] Unclosed pyproject.toml leads to a ResourceWarning

### ISSUE THREAD

**Issue by lociii** [Riverside-Healthcare/djLint#469]
Sorry for "abusing" the feature request template but there is non that fits bugs related to the library implementation, not the formatter or linter.

- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## Library bug report

```
$ djlint template.html 
/home/app/venv/lib/python3.10/site-packages/djlint/settings.py:119: ResourceWarning: unclosed file <_io.BufferedReader name='/app/pyproject.toml'>
content = tomllib.load(pyproject_file.open("rb"))
```

The current way of loading the pyproject.tml file does not properly close the file descriptor.  
See https://github.com/Riverside-Healthcare/djLint/blob/master/src/djlint/settings.py#L119

Therefore 
```python
with open(pyproject_file. "rb") as fp:
    content = tomllib.load(fp)
```
should solve the issue and not trigger a warning anymore.  
I'd be happy to supply a merge request if the solution is accepted.

**Comment by christopherpickering** [comment]
Nice catch! Since its a path object, maybe we can do `.read_bytes()` to keep it inline?

**Comment by lociii** [comment]
Awesome. Thanks for the fix!
